appinfo/routes.php

```diff
@@ -31,47 +31,79 @@
 		['name' => 'page#vote_public', 'url' => '/poll/{token}', 'verb' => 'GET', 'postfix' => 'oldpublic'],
 
 		['name' => 'subscription#get', 'url' => '/subscription/get/{pollId}', 'verb' => 'GET'],
```

Comment:
I'd remove the word `get` from GET requests, as it's clear this is a getter. But I'm fine if you think otherwise :)